spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/ReactorLoadBalancerExchangeFilterFunction.java

```diff
@@ -77,13 +80,21 @@ public ReactorLoadBalancerExchangeFilterFunction(ReactiveLoadBalancer.Factory<Se
 		this(loadBalancerFactory, properties, Collections.emptyList());
 	}
 
+	@Deprecated
```

Comment:
Please add javadoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added